app/riot/.babelrc

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Are these configs very different from the one in the root?

Can we use the root's .babelrc ?